src/components/Calculator.js

```diff
@@ -1,146 +1,139 @@
             </div>
-            <div className="operators">
+            <div className="numbers">
+              <button type="button" onClick={handleClick} className="key zero" value={0}> 0</button>
+              <button
+                type="button"
+                onClick={handleClick}
+                className="key dot"
+                value="."
+              >
+                .
+              </button>
+              <button
+                type="button"
+                onClick={handleClick}
+                className="key one"
+                value={1}
+              >
+                1
+              </button>
+              <button
+                type="button"
+                onClick={handleClick}
+                className="key two"
+                value={2}
+              >
+                2
+              </button>
               <button
                 type="button"
-                className="key divide"
+                onClick={handleClick}
+                className="key three"
+                value={3}
               >
-                /
+                3
               </button>
               <button
                 type="button"
-                className="key multiply"
+                onClick={handleClick}
+                className="key four"
+                value={4}
               >
-                *
+                4
               </button>
               <button
                 type="button"
-                className="key subtract"
+                onClick={handleClick}
+                className="key five"
+                value={5}
               >
-                -
+                5
               </button>
               <button
                 type="button"
-                className="key add"
+                onClick={handleClick}
+                className="key six"
+                value={6}
               >
-                +
+                6
               </button>
-              <button type="button" className="key equalTo">
-                =
+              <button
+                type="button"
+                onClick={handleClick}
+                className="key seven"
+                value={7}
+              >
+                7
+              </button>
+              <button
+                type="button"
+                onClick={handleClick}
+                className="key eight"
+                value={8}
+              >
+                8
+              </button>
+              <button
+                type="button"
+                onClick={handleClick}
+                className="key nine"
+                value={9}
+              >
+                9
               </button>
             </div>
           </div>
+          <div className="operators">
+            <button type="button" onClick={handleClick} className="key divide" value="÷">/ </button>
+            <button type="button" onClick={handleClick} className="key multiply" value="*">* </button>
```

Comment:
- [ ] Your multiplication here is not working. Please change the value from '*' to 'x' for it to work

Reply:
Thank you for the review @Lucas-Masaba . Let me do the needful

Reply:
well resolved! ✔️